src/common/unit_system.h

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023 Dmitry Kalinkin
+
+#pragma once
+
+namespace eicrecon {
+
+  // This allows to to have a trial of https://github.com/eic/EDM4eic/pull/35
+  // Shoult it be a success, we will re-export the edm4hep::unit namespace from
+  // here.
+
+  namespace unit {
+
+    //
+    // This provides definition for a unique unit system for this EDM. It is
+    // neither Geant, nor TGeo. The DD4hep, by default, uses a system matching
+    // the TGeo one.
+    //
+    // +------------+---------------------|-----------------+
+    // | Geant      | TGeo                | EDM4hep/EDM4eic |
+    // +------------+---------------------|-----------------+
+    // | milimeter  | centimeter          | milimeter       |
+    // | nanosecond | second              | nanosecond      |
+    // | MeV        | GeV                 | GeV             |
+    // +------------+---------------------|-----------------+
+    //
+
+    // distance
+    static constexpr double mm = 1.0; // millimeter
+    static constexpr double cm = 1e1 * mm; // centimeter
+    static constexpr double micrometer = 1e-3 * mm; // micrometer
```

Comment:
Good to have both, because some write "u" to mean Greek letter "mu" (`dd4hep::micrometer == dd4hep::um`).
```suggestion
    static constexpr double um = 1e-3 * mm; // micrometer
    static constexpr double micrometer = um;
```

Reply:
I like the idea for mu -> u. Implemented.